cmd/operator/README.md

```diff
@@ -15,6 +15,42 @@ administrators.
 
 ## Installation
 
+Install NFD to perform node labelling:
+
+```
+$ kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref=v0.10.1
+```
+Make sure both NFD master and worker pods are running:
+
+$ kubectl get pods -n node-feature-discovery
+NAME                          READY   STATUS    RESTARTS   AGE
+nfd-master-599c58dffc-9wql4   1/1     Running   0          25h
+nfd-worker-qqq4h              1/1     Running   0          25h
+
+Deploy labeling rules:
+
+$ kubectl apply -f https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/node-feature-rules.yaml
```

Comment:
GPU and SGX do not work correctly with the default deployment. could we move `sgx_nfd` as part of `deployments/nfd` (and the GPU equivalent `gpu.intel.com`) too?

Reply:
@mythi thank you for the review! Done.

Reply:
@bart0sh I'd suggest we do so that the `ref=v0.10.1` is hidden from the documentation and end users and we add something like:

```
$ kubectl apply -f https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd
```

with `kustomization.yaml`:
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
bases:
- "https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref=v0.10.1"
resources:
- node-feature-rules.yaml
```

I can then move the SGX stuff as an overlay:
```
$ kubectl apply -f https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/sgx
```

Reply:
done